ArduPlane/quadplane.h

```diff
@@ -76,6 +76,14 @@ class QuadPlane
     void control_auto(void);
     bool setup(void);
 
+    enum class ThrustType : uint8_t {
+        THRUST_TYPE_UNDEFINED=0, // Undefined. Means that no backend has configured it yet.
```

Comment:
```suggestion
        UNDEFINED=0, // Undefined. Means that no backend has configured it yet.
```

superfluous.  similarly on other entries

Reply:
Amended.